### DISCUSSION TITLE
To play MOD files on a c128

### DISCUSSION THREAD

**Discussion by odflor** [frntc/Sidekick64#70] [General]
Hi Pals!

I copy mod music files on the sd to play with SK64, but the files does not appear on the file browser, even if I put it on the SID folder.

1) How I do to play these files?
2) To do _"audio player (MOD, YM and WAV files) with SID- and HDMI-output, supporting up to stereo 48kHz output on dual-SID setups (C64 and C128)"_, must I wire the SK64?

Best Regards

**Comment by pigravity** [comment]
1. put in Music folder.
2. afaik, no.

**Comment by odflor** [comment]
I create the "MUSIC" folder on the sd root, but it doesn't show on the F7 menu.

Then I move the "MUSIC" folder into the "SID" folder and I can see the folder, but have not any content.

The folder MUSIC is plently of Amiga .MOD files and .YM files to.

How I can do to play .MOD files using SD64?

**Reply by frntc** [reply]
It should exactly work like that: copy to MUSIC folder, start playing with RETURN (for normal display), SPACE (for color bars), and SHIFT+RETURN/SPACE for playing via HDMI.

**Reply by odflor** [reply]
My problem is that none of .mod files appears on the menu (F7) to press RETURN over them.

What I need to do to allow SD64 to show me the .mod files in the file browser?

**Reply by frntc** [reply]
I understood and just wanted to confirm that you're trying it the correct way. Maybe something's wrong with your SD-card? (reformat and copy all files again for testing)

**Reply by odflor** [reply]
I override the contents of my SD with a new version and works. I lose all my favorites preferences, but finally I can ear MOD and YM files!
